.flake8

```diff
@@ -5,7 +5,30 @@ import-order-style = pycharm
 max-line-length = 127
 max-complexity = 10
 
-select=E9,E712,F63,F7,F82,W605,T001,T002,T003,T004
+ignore=
```

Comment:
Can we get a link in here to a list with error code details. Maybe to https://flake8.pycqa.org/en/latest/user/error-codes.html ? 

I hate these long lists of error codes. It would be great to be able to easily see what they are.

I think it makes sense to explicitly be able to see what is being ignored, it's probably more relevant than being able to see what is enabled but no idea what is being skipped.

Reply:
And link to https://pep8.readthedocs.io/en/latest/intro.html#error-codes also